jsonnet/rules.jsonnet

```diff
@@ -196,6 +196,17 @@
               severity: 'warning',
             },
           },
+          {
+            expr: 'sum(max by(namespace, container, pod) (increase(kube_pod_container_status_restarts_total[12m])) and max by(namespace, container, pod) (kube_pod_container_status_last_terminated_reason{reason="OOMKilled"}) == 1) > 10',
+            alert: 'MultipleContainersOOMKilled',
+            'for': '15m',
+            annotations: {
+              message: 'Multiple containers were out of memory killed within the past 15 minutes.',
+            },
+            labels: {
+              severity: 'info',
```

Comment:
Why `info`? OOM means we don't have enough memory in a cluster and it should be scaled, which in many cases is a manual action requiring human intervention and possibly a page. Can we raise it to `warning`?

Reply:
No, because this alert is not exact we do not want to do warning level, decided in the PR that was merged and offline conversations with the team. They are adding a proper metric in crio directly in 4.5 that will actually have the info we need on how many containers are being OOM and we will remove this alert and make it warning then. For now as this is good enough alert and its mainly used for us to get a ballpark on how often we see OOMs in the wild.

See https://github.com/openshift/cluster-monitoring-operator/pull/668 and https://github.com/openshift/cluster-monitoring-operator/pull/642#discussion_r375254585

Reply:
It's mostly because this is a heuristic, we assume that if a container restarted and it's last termination reason was an OOM, then all of those restarts were OOMs. If we truly knew that x amount of OOMs happened then I would agree to making this warning but with this heuristic I would be more careful.